### PR TITLE
chore: Prepare for 0.18.4

### PR DESCRIPTION
**Pull request by CsCherrYY**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,14 @@ All notable changes to the "vscode-java-dependency" extension will be documented
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## 0.18.4
+### Added
+- Show reports when exporting jar. [#374](https://github.com/microsoft/vscode-java-dependency/issues/374)
+- Extension is unavailable in virtual workspaces currently. [#490](https://github.com/microsoft/vscode-java-dependency/issues/490)
+- Extension may require your trust on workspace for code execution. [PR#486](https://github.com/microsoft/vscode-java-dependency/pull/486)
```
Comment (jdneo) [inline CHANGELOG.md:11]: @Eskibear Need your input here. We add fields about trust workspace in package.json. Is there any behavior change users can notice??

Comment (Eskibear) [reply CHANGELOG.md:11]: >  Extension may require your trust on workspace for code execution. 

I don't think we should mention this in release note. As "workspace Trust" feature hasn't been release by vscode yet. We can wait until vscode ship it.

Comment (Eskibear) [reply CHANGELOG.md:11]: BTW workspace trust API has changed, I'm updating it now.

Comment (Eskibear) [reply CHANGELOG.md:11]: Please review this PR first https://github.com/microsoft/vscode-java-dependency/pull/501, before you release the extension

**Review by jdneo — APPROVED** [review]

**Pull request merged**